src/components/InForm/InForm.tsx

```diff
@@ -32,42 +33,51 @@ export const InForm: React.FunctionComponent<any> = (props) => {
   const [formData, setFormData] = useState<IInForm>(defaultInForm)
```

Comment:
Line above this -- defaultInForm should probably be a const as well (instead of let).

Reply:
Completely agree, also go rid of my TODO comment that is done